components/patient-access-rules/api/src/main/java/org/phenotips/data/permissions/script/PermissionsManagerScriptService.java

```diff
@@ -92,13 +107,29 @@ public AccessLevel resolveAccessLevel(String name)
         return this.manager.resolveAccessLevel(name);
     }
 
-    public PatientAccess getPatientAccess(Patient targetPatient)
+    public PatientAccess getPatientAccess(String targetPatientId)
```

Comment:
The old method should be left in place, too, for backwards compatibility, and it can just do `return getPatientAccess(serializer.serialize(targetPatient.getDocumentReference());`